src/BenchmarksApps/TLS/Kestrel/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# This stage is used when running from VS in fast mode (Default for Debug configuration)
+FROM mcr.microsoft.com/dotnet/aspnet:9.0-alpine AS base
```

Comment:
base on azure mariner base image (linux vms)

Reply:
discussed offline; should not expand here and we will create a separate job to run based on azurelinux image